### ISSUE TITLE
Handle COBOL keywords in SPECIAL-NAMES CLASS definition

### ISSUE THREAD

**Issue by rooksdo** [TypeCobolTeam/TypeCobol#1895]
### Description
An exception is fired when declaring a `SPECIAL-NAME` class with values `SPACE`, `QUOTE`, `LOW-VALUE`, `HIGH-VALUE`, `ZERO`... :

```
Multiple markers at this line
- Runtime error: System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet.
   à TypeCobol.Compiler.Parser.CodeElementBuilder.CreateCharacterInCollatingSequence(CharacterInCollatingSequenceContext context)
   à TypeCobol.Compiler.Parser.CodeElementBuilder.CreateCharactersRange(CharactersRangeContext context)
   à TypeCobol.Compiler.Parser.CodeElementBuilder.EnterSpecialNamesParagraph(SpecialNamesParagraphContext context)
   à TypeCobol.Compiler.Parser.Generated.CodeElementsParser.SpecialNamesParagraphContext.EnterRule(IParseTreeListener listener)
   à Antlr4.Runtime.Tree.ParseTreeWalker.EnterRule(IParseTreeListener listener, IRuleNode r)
   à Antlr4.Runtime.Tree.ParseTreeWalker.Walk(IParseTreeListener listener, IParseTree t)
   à TypeCobol.Compiler.Parser.CodeElementsParserStep.ParseProcessedTokensLinesChanges(TextSourceInfo textSourceInfo, ISearchableReadOnlyList`1 documentLines, IList`1 processedTokensLinesChanges, PrepareDocumentLineForUpdate prepareDocumentLineForUpdate, TypeCobolOptions compilerOptions, PerfStatsForParserInvocation perfStatsForParserInvocation)
```
**Sample test**
```cob
       IDENTIFICATION DIVISION.
       PROGRAM-ID. Pgm.
       ENVIRONMENT DIVISION.
       CONFIGURATION SECTION.
       SPECIAL-NAMES. DECIMAL-POINT IS COMMA
      *    same exception with SPACE QUOTE LOW-VALUE HIGH-VALUE ZERO
           CLASS VALID-NAME IS SPACE
           .
       END PROGRAM Pgm.
```

**Comment by fm-117** [comment]
Could you check with code from branch [317_SpecialNamesParagraph](https://github.com/TypeCobolTeam/TypeCobol/tree/317_SpecialNamesParagraph) ? Because code has been changed to improve support of special names declarations.

**Comment by rooksdo** [comment]
@fm-117 
Seems that the bug is still present on branch [317_SpecialNamesParagraph](https://github.com/TypeCobolTeam/TypeCobol/tree/317_SpecialNamesParagraph)

**Comment by fm-117** [comment]
Looks like it's another case where figurative constants are allowed but not listed in our g4 grammar. The fix for this should be somewhat similar to #317.
